doc/LabMeetings.rst

```diff
@@ -0,0 +1,47 @@
+Gymrek Lab Meetings
+===================
+
+General meeting info
+---------------------
+
+The lab meeting schedule is emailed out at the beginning of each quarter to our Google Group (see :ref:`onboarding <onboarding>` to request access).
+
+We cater food for each lab meeting.
+Each week, someone in the lab will be assigned to present. The following week, they will usually be assigned to order food for the lab.
+Refer to :ref:`these instructions <food>` for ordering food if you've been assigned.
+
+.. _labmeetings-presenting:
+
+Presenting
+----------
+If you are presenting in Franklin Antonio Hall, follow these steps to set up the projector for your presentation.
+
+1. It's best to connect to the projector via the cables provided in the front of the room - rather than connecting wirelessly.
+    You should connect **both** the HDMI cable and the USB cable to your laptop. The USB cable lets you connect to the room's speaker and microphone.
+    If needed, you can use the white USB Type-C adapter to connect to both cables like `this <https://drive.google.com/file/d/1CItoMXvJZ3ulQaZqI65ScLRaIbQMhJwX>`_.
+
+    .. figure:: https://drive.google.com/uc?export=view&id=1CItoMXvJZ3ulQaZqI65ScLRaIbQMhJwX
+        :alt: Proper dongle setup
+        :align: center
+        :width: 400px
+
+2. Configure your laptop for presentation mode. See these `instructions for Macs <https://support.apple.com/guide/mac-help/extend-mirror-mac-desktop-multiple-displays-mchlb5f905a1/mac>`_ or simply click *Windows Key + P* together on Windows.
+3. Since all lab meetings are hybrid, you should run zoom during the meeting. Log into the lab's zoom meeting now. You can find the Zoom link on the lab meeting schedule.
+4. Display the Zoom window via the projector. `Screenshare your slides from Zoom <https://support.zoom.us/hc/en-us/articles/201362153-Sharing-your-screen-or-desktop-on-Zoom#top1>`_.
+5. Connect your Zoom video to the room's camera. In FAH 4002, you can select the second of the *"USB Capture SDI+"* options.
+
+    .. figure:: https://drive.google.com/uc?export=view&id=1ynzFmmV0UMNqKNkzVpfiHPi5heDyYIv4
+        :alt: Proper dongle setup
+        :align: center
+        :width: 400px
+
+6. Connect your Zoom microphone to the room's microphone. In FAH 4002, you can select *"Line In (USB Capture SDI+ Line In)"* from among the list of options. You might want to double-check that someone on zoom can hear someone else on the opposite side of the room.
+
+    .. figure:: https://drive.google.com/uc?export=view&id=1l8h0EdrPTsMATmPFfqnWukNf-J2L_6rB
+        :alt: Proper dongle setup
+        :align: center
+        :width: 400px
+
+7. Connect your Zoom audio to the room's speakers. In FAH 4002, you can select *"Creston (Intel(R) Display Audio)"* from among the list of options. Have someone else log into the Zoom and say something through their device.
```

Comment:
Should be Crestron under "Creston (Intel(R) Display Audio)"

Reply:
Ooh, good catch

```suggestion
7. Connect your Zoom audio to the room's speakers. In FAH 4002, you can select *"Crestron (Intel(R) Display Audio)"* from among the list of options. Have someone else log into the Zoom and say something through their device.
```